Rename the *_env_blacklist config opts to *_saltenv_blacklist

terminalmage opened the pull request:
These were overlooked a while back when we deprecated env in favor of
saltenv.

terminalmage committed:
Rename the *_env_blacklist config opts to *_saltenv_blacklist

These were overlooked a while back when we deprecated env in favor of
saltenv.

salt-jenkins commented:
@terminalmage, thanks for your PR! By analyzing the history of the files in this pull request, we identified @gtmanfred, @rallytime and @thatch45 to be potential reviewers.

terminalmage committed:
Remove space before comma to satisfy linter

terminalmage commented:
Unit tests need to be updated before this can be merged.

terminalmage committed:
Update fileserver unit tests

This fixes the tests by adding back gitfs_env_blacklist, and also
converts the config tests to use the new LoaderModuleMixin.

cachedout commented on salt/version.py:
```diff
@@ -96,107 +96,107 @@ class SaltStackVersion(object):
         'Nitrogen'      : (MAX_SIZE - 102, 0),
         'Oxygen'        : (MAX_SIZE - 101, 0),
         'Fluorine'      : (MAX_SIZE - 100, 0),
+        'Neon'          : (MAX_SIZE - 99, 0),
```
I'm not following how this change is related to the purpose of the PR?

terminalmage replied:
Without it, the calls to ``salt.utils.warn_until()`` I added will throw a ValueError since ``Neon`` is not defined in version.py. For example, try with a much later release codename:

```python
>>> import salt.utils
>>> salt.utils.warn_until('Lead', 'Message')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/erik/git/salt/main/salt/utils/__init__.py", line 2227, in warn_until
    version = salt.version.SaltStackVersion.from_name(version)
  File "/home/erik/git/salt/main/salt/version.py", line 264, in from_name
    'Named version \'{0}\' is not known'.format(name)
ValueError: Named version 'Lead' is not known
```

terminalmage replied:
Just FYI, this is something we have to do each release cycle so that we can have release n+2 supported in ``salt.utils.warn_until()``.

rallytime replied:
Yeah, we do need the `Neon` line uncommented for this change for the `warn_until`. But git is being silly by changing _all_ of those lines...

terminalmage replied:
All of what lines? Do you mean the ones that are commented out? Those all needed to be commented out because the linter does not like whitespace after arguments. I noticed this for Neon because it is the first one with a two-digit number in it, and decided to fix all of the other ones as well to avoid lint errors every time we have to uncomment a version.

rallytime replied:
Ah, i was hard to tell what was different, but I see what you mean.
